releng/tools.mdsd.ecoreworkflow.targetplatform/tp.target

```diff
@@ -1,5 +1,9 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <?pde version="3.8"?><target name="tools.mdsd.ecoreworkflow Target Platform" sequenceNumber="1">
 <locations>
+    <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit" refresh="true">
```

Comment:
Did you intentionally use the refresh feature? The version on the release update site will always be the same.

Reply:
No, my bad. C/P mistake.